bubble_sort.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+def bubble_sort(arr)
+  length = arr.length
+  for i in 0..length - 1 do
+    for j in 0..length-i-2 do
+      if arr[j] > arr[j + 1]
+        carry = arrToSort[j + 1]
```

Comment:
**[REQUIRED]**  `arrToSort` variable is not defined. After changing it to `arr` the code passes the sample test cases. Let's fix this